exercises/1/webapp/src/components/Home.vue

```diff
@@ -0,0 +1,171 @@
+<template>
+  <div id="app">
+    <div class="news-list">
+      <div class="news-list-wrapper">
+        <h1>{{msg}}</h1>
+        <div class="news-holder" v-for="(newsContent, index) in sortedNewsList" :key="index">
+          <News :newsTitle="newsContent.title" :newsBody="newsContent.body" :newsVotes="newsContent.votes" @updateItem="mewsUpdateMessage" @removeItem="newsRemoveMessage"/>
```

Comment:
```suggestion
          <News :news="news" @updateItem="mewsUpdateMessage" @removeItem="newsRemoveMessage"/>
```

Why not pass the entire object? :thinking:

Reply:
I've just fixed it